docker/docker.mk

```diff
@@ -116,28 +118,29 @@ docker-build: check-docker-env check-arch
 	$(info Building $(DOCKER_IMAGE):$(DAPR_TAG) docker images ...)
 ifeq ($(TARGET_ARCH),$(TARGET_ARCH_LOCAL))
 ifeq ($(ONLY_DAPR_IMAGE),true)
-	$(DOCKER) build --build-arg PKG_FILES=* $(BUILD_ARGS) -f $(DOCKERFILE_DIR)/$(DOCKERFILE) $(BIN_PATH) -t $(DOCKER_IMAGE):$(BUILD_TAG)
+	$(DOCKER) build --output type=docker --build-arg PKG_FILES=* $(BUILD_ARGS) -f $(DOCKERFILE_DIR)/$(DOCKERFILE) $(BIN_PATH) -t $(DOCKER_IMAGE):$(BUILD_TAG)
```

Comment:
For some reason, on my desktop, the image was generated as an OCI image and was a manifest list, with the actual image and an "unknown" entry for OS and architecture. The mimetype was OCI but forcing it to docker fixed it. All the current images we publish are docker images. This might be just on my desktop but forcing this will make this fool proof. The buildx commands worked fine and did not need this flag.

Reply:
the MAC desktop?

Reply:
Yes, MacOS.